README.md

```diff
@@ -142,13 +151,13 @@ Requirements for parameters of generation process:
 * <i>run_parallel</i> - data type - boolean, default value is False
 * <i>batch_size</i> - data type - integer, must be equal to or more than 1
 * <i>random_seed</i> - data type - integer, must be equal to or more than 0
-* <i>print_report</i> - data type - boolean, default value is False
+* <i>reports</i> - data type - if the value is passed through CLI - string, if the value is passed in the metadata file - string (if value is <i>"none"</i> or <i>"all"</i>) or list, accepted values: <i>"none"</i> (default) - no reports will be generated, <i>"all"</i> - generates an accuracy report, <i>"accuracy"</i> - generates an accuracy report, <i>"metrics_only"</i> - outputs the metrics information only to standard output. Default value is <i>none</i>. In the metadata file multiple values can be specified as a list to generate multiple types of reports simultaneously
```

Comment:
Sorry, doesn't know how to pass list in yaml files. Just a comma separated list? Or separate entries with the same name and different values?